backend/routes/users.js

```diff
@@ -1,17 +1,30 @@
-const express = require('express');
-const users = require('../logic/users/users');
+const express = require("express");
+const users = require("../logic/users/users");
+const curriculum = require("../logic/courses/curriculum");
 const router = express.Router();
 
 /* GET users listing. */
-router.post('/addStudentUser', function(req, res, next) {
-	let username = req.query.username;
-	let password = req.query.password;
-	let email = req.query.email;
-	let id = req.query.student_id;
-	users.insertStudentUser(username, password, email, id)
-	.then(function(val) {
-		console.log(val);
-		res.send(val);
+router.post("/addStudentUser", function(req, res, next) {
+  let username = req.body.username;
+	let password = req.body.password;
+	let email = req.body.email;
```

Comment:
Formatting is broken

Reply:
once we fixed this formatting we can merge this